docs/after/checkup.md

```diff
@@ -1,35 +1,33 @@
-
-### Things to prepare
-Several things to take note: 
-- Bring your documents and items required for the medical checkup. 
-- Your passport
-- Medical history/reports written in English (if any)
-- Cash or NETS card
-- [NUS Medical Examination Form](http://www.nus.edu.sg/uhc/docs/default-source/default-document-library/undergraduate.pdf?sfvrsn=62ea6b98_2) with **Part I completed**
-- Prescription eyewear / contact lens
-- Wear a plain t-shirt or blouse (but if you don’t have one, it’s okay, usually they’ll ask you to wear their disposable X-ray gown anyway).
-- **For girls**, please do the medical examination at least 5 to 7 days after the end of your menses, since there will be a urine test. However, if that’s not possible they usually allow you to take the urine test some other time.
-
-### Getting there
-:::note
-The medical examination will be at the University Health Centre (UHC), NUS (_it’s not in NUH :)_) 
-:::
-
-To get there, you can take the NUS internal shuttle bus. For bus lines A1 / D2, alight at the _Opposite University Health Centre_ bus stop while for bus lines A2 / C, alight at the _University Health Centre_ bus stop.
-
-### Next steps
-
-One week after your check up, don’t forget to bring your passport and go back to UHC to collect your results                                  
+---
+id: checkup
+title: Checkup
+---
+
+We strongly encourage you to conduct your medical examination in NUS to avoid any unwanted complications and incompatibilities that can arise if you complete the test in Indonesia. Please **do not delay** in taking this test because you will need it for the issuance of Student Pass, therefore, we recommend you to take it before QET or the day after.
+
+For the latest information, please refer [here](http://www.nus.edu.sg/uhc/general-health/medical-examination/pre-admission/pre-admission-medical-exam)
+
+### Booking 
+Early booking is essential to get the date you want! [Book your appointment here](https://www.gevme.com/nus-pre-admission-medical-examination-2020). It will cost $55 (fifty-five Singapore dollars) for you to get the test done here.
+
+### Things to prepare
+Several things to take note: 
+- Bring your documents and items required for the medical checkup. 
+- Your passport
+- Medical history/reports written in English (if any)
+- Cash or NETS card
+- [NUS Medical Examination Form](http://www.nus.edu.sg/uhc/docs/default-source/default-document-library/undergraduate.pdf?sfvrsn=62ea6b98_2) with **Part I completed**
+- Prescription eyewear / contact lens
+- Wear a plain t-shirt or blouse (but if you don’t have one, it’s okay, usually they’ll ask you to wear their disposable X-ray gown anyway).
+- **For girls**, please do the medical examination at least 5 to 7 days after the end of your menses, since there will be a urine test. However, if that’s not possible they usually allow you to take the urine test some other time.
```

Comment:
```suggestion
- **For girls**, please do the medical examination at least 5 to 7 days after the end of your period, since there will be a urine test. However, if that’s not possible they usually allow you to take the urine test some other time.
```